From 94db5061bb2a729b6736e264de9eecb6295a225b Mon Sep 17 00:00:00 2001
From: Richard Zhu <hongxing.zhu@nxp.com>
Date: Wed, 9 May 2018 08:37:07 +0800
Subject: [PATCH 3743/5242] MLK-18238 ARM64: dts: imx8qm: add pd for rpmsg to
 avoid crash

commit  7026163ec34de91fa46ab2d08cfece49c11387c1 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add pd for rpmsg to avoid crash, after enable
xrdc blocking.

Signed-off-by: Richard Zhu <hongxing.zhu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-device.dtsi      |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
index baf9f0b..915f89a 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
@@ -3605,6 +3605,7 @@
 
 		rpmsg: rpmsg {
 			compatible = "fsl,imx8qm-rpmsg";
+			power-domains = <&pd_cm40_mu0a0>;
 			status = "disabled";
 		};
 
@@ -3622,6 +3623,7 @@
 		rpmsg1: rpmsg1{
 			compatible = "fsl,imx8qm-rpmsg";
 			multi-core-id = <1>;
+			power-domains = <&pd_cm41_mu0a0>;
 			status = "disabled";
 		};
 	};
-- 
1.7.9.5

